core/templates/domain/utilities/url-interpolation.service.ts

```diff
@@ -115,11 +115,10 @@ export class UrlInterpolationService {
    */
   interpolateUrl(
       urlTemplate: string,
-      interpolationValues: InterpolationValuesType): string | null {
+      interpolationValues: InterpolationValuesType): string {
     if (!urlTemplate) {
       this.alertsService.fatalWarning(
         'Invalid or empty URL template passed in: \'' + urlTemplate + '\'');
```

Comment:
Function `this.alertsService.fatalWarning` already throws an error and shows a warning thereby stopping function execution. `return null` statements are redundant for this function